### PR TITLE
feat(profiling): add wizard onboarding docs for rust

### PR DESCRIPTION
**Pull request by viglia**
This PR adds "wizard" docs for Rust profiling.

### CODE REVIEW

File: src/wizard/rust/profiling-onboarding/rust/1.install.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+type: language
+---
+
+#### Install
+
+For the Profiling integration to work, you must have the Sentry Rust SDK crate (minimum version 0.29.0). Learn more about installation methods in our [full documentation](https://docs.sentry.io/platforms/rust/#install).
+
+```cargo
+[dependencies]
+sentry = { version = "0.29.0", features = ["profiling"] }
```
Comment (Swatinem) [inline src/wizard/rust/profiling-onboarding/rust/1.install.md:14]: I believe we have some magic snippet that automatically fills in the latest version. Other examples / wizards should have that.

File: src/wizard/rust/profiling-onboarding/rust/2.configure-performance.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+#### Configure Performance
+
+Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:
+
+```rust
+sentry::init((
+    "___DSN___",
+    sentry::ClientOptions {
+        traces_sample_rate: 1.0,
```
Comment (Swatinem) [inline src/wizard/rust/profiling-onboarding/rust/2.configure-performance.md:16]: I believe all the docs should also configure a `release`, right?

File: src/wizard/rust/profiling-onboarding/rust/0.alert.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+name: Rust
+doc_link: https://docs.sentry.io/platforms/rust/profiling/
+support_level: alpha
+type: language
+---
+
+<div class='alert warning'>
+Profiling for Rust is currently in Alpha, and there may be some bugs. We recognize the irony. If you have any questions or feedback, join our <a href="https://discord.gg/zrMjKA4Vnz">discord channel</a>.
```
Comment (Swatinem) [inline src/wizard/rust/profiling-onboarding/rust/0.alert.md:9]: Is there some inside joke about irony here that I don’t know yet? :-D

File: src/wizard/rust/profiling-onboarding/rust/1.install.md (hunk 1 above)
Comment (viglia) [reply src/wizard/rust/profiling-onboarding/rust/1.install.md:14]: nice, let me fix this then! :)

File: src/wizard/rust/profiling-onboarding/rust/2.configure-performance.md (hunk 2 above)
Comment (viglia) [reply src/wizard/rust/profiling-onboarding/rust/2.configure-performance.md:16]: I didn't know about this requirement for docs. I'll add it to align with the rest then.

File: src/wizard/rust/profiling-onboarding/rust/0.alert.md (hunk 3 above)
Comment (viglia) [reply src/wizard/rust/profiling-onboarding/rust/0.alert.md:9]: This is a "template" warning we're using across all our SDKs that are currently in alpha.

The irony lies in the fact that we're using a profiler to catch performance issue and/or bugs, but the profiler itself being in alpha could have bugs/perf. issues. I find it pretty ironic indeed.

**Review by Swatinem — APPROVED** [review]

**Pull request merged**